src/pipeline.cc

```diff
@@ -1404,6 +1405,10 @@ Napi::Value pipeline(const Napi::CallbackInfo& info) {
     vips_enum_from_nick(nullptr, VIPS_TYPE_FOREIGN_DZ_DEPTH,
     sharp::AttrAsStr(options, "tileDepth").data()));
 
+  if (sharp::HasAttr(options, "centre")) {
+    baton->centre = sharp::AttrAsBool(options, "centre");
```

Comment:
The `if` wrapper can be removed by ensuring `this.options.tileCentre` defaults to `false` in JS land.